cdap-ui/app/cdap/main.js

```diff
@@ -96,6 +100,8 @@ class CDAP extends Component {
       version: '',
       authorizationFailed: false,
       loading: true,
+      pageLevelError: false,
+      retrievingNs: false,
```

Comment:
Have simple variable name. `retrieving` sounds like its still doing it. `isNamespaceFetchInFlight` may be?

Reply:
I feel this is similar to other variables we use like `loading` for example and `retrievingNs` sounds simpler than `isNamespaceFetchInFlight`, no? :sweat_smile:

Reply:
IMHO `isNamespaceFetchInFlight` says exactly what is the variable tracking. If you feel `retrievingNs` makes sense we can have it. Don't want to stall the PR over a variable name.